config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1338,4 +1338,32 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-kubelet
     testgrid-tab-name: kubelet-credential-provider
-    description: "tests alpha feature kubelet image credential provider"
+    description: "tests feature kubelet image credential provider"
+- name: ci-kubernetes-e2e-gcp-credential-provider
+  interval: 6h
```

Comment:
we probably don't need this running 4 times a day... not sure if sig-testing has a recommendation for frequency on super-targeted jobs like this... maybe 24h?

also, can we make an optional pull-request version of this job we can trigger prior to merge?

Reply:
+1 on pre-submit job as well, see `pull-kubernetes-node-kubelet-credential-provider` for an example

Reply:
done, PTAL